utility/get_ceph_secret.py

```diff
@@ -0,0 +1,19 @@
+import subprocess
+import shlex
+from ocs import api_client as ac
+
+
+def get_ceph_secret(namespace='openshift-storage'):
+    client = ac.get_api_client("OCRESTClient")
+    pod_list = client.get_pods(namespace=namespace)
+    toolpod = [x for x in pod_list if '-tools-' in x][0]
+    ceph_cmd = f"oc -n openshift-storage exec {toolpod}"
+    secret_cmd = f"{ceph_cmd} -- ceph auth get-key client.admin"
```

Comment:
Since storageclass.yaml has "userid:kubernetes" enabled, we also need to get the key for "client.kubernetes" from tookbox pod.
e.g.
secret_cmd_user = f"{ceph_cmd} -- ceph auth get-key client.kubernetes"

E.g. here - https://github.com/rook/rook/blob/master/Documentation/ceph-csi-drivers.md